clang-tools-extra/docs/clang-tidy/checks/performance/redundant-lookup.rst

```diff
@@ -0,0 +1,147 @@
+.. title:: clang-tidy - performance-redundant-lookup
+
+performance-redundant-lookup
+============================
+
+This check warns about potential redundant container lookup operations within
```

Comment:
Please synchronize with statement in Release Notes.

Reply:
Fixed in d9d3984a056192b8ef61f385090d90c55774123d